packages/sdk/package.json

```diff
@@ -53,6 +53,7 @@
     "@libp2p/websockets": "^5.0.3",
     "@waku/relay": "0.0.1",
     "@waku/core": "0.0.18",
+    "@waku/utils": "*",
```

Comment:
I think we agreed on fixed version everywhere. The presence of wildcards was because release-pleaes did not yet release said package.

Reply:
Yes, that's also my expectation.

Reply:
Should be the case but also it can be set manually - no warning should happen.